src/backends.jl

```diff
@@ -135,7 +135,7 @@ function pickDefaultBackend()
             Pkg.installed(env_default)  # this will error if not installed
             sym = Symbol(lowercase(env_default))
             if haskey(_backendType, sym)
-                return backend(sym)
+                return @eval backend($sym)
```

Comment:
> Hm, I do not see that You have set PLOTS_DEFAULT_BACKEND=GR but it is not installed. warning anywhere

Yes, I have specified GR as my local default backend in my `.juliarc` with `ENV["PLOTS_DEFAULT_BACKEND"] = "GR"`. If you have not explicitly defined your default backend, you don't end up here.
You can reproduce the issue I described with:
```julia
using Plots
sym = :gr
@eval backend($sym)
```
`@eval backend($sym())` would work though.

> And I'm also not seeing any world age issues anymore, but that might just be because of the Atom.jl PR that makes use of invokelatest. Or did that error appear for anyone not using Juno?

Oh, you are right! I was not aware that this was only in Juno, because I was only using Juno when testing. Are the world age issues now fixed in Atom.jl even without the `@eval`s here?